sciencebeam-convert.sh

```diff
@@ -16,11 +16,9 @@ if [ ! -f "${host_dir}/${sample_filename}" ]; then
   wget $download_url --output-document=${host_dir}/${sample_filename}
 fi
 
-image_name=elifesciences/sciencebeam
-
 container_dir=/home/sciencebeam/data
 
-docker run --net sciencebeam-texture -v ${host_dir}:${container_dir} ${image_name} \
+docker-compose run sciencebeam \
```

Comment:
this is running a separate container usually called `sciencebeam_run_...` which is not the one in the `services:` definition. May want to add a `--rm` to delete the container when it has finished running